### PR TITLE
[#76] Minor improvements

### PR DESCRIPTION
**Pull request by Kyrrui**

### CODE REVIEW

File: universal-login-example/test/services/BackupService.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    identityService = [];
+    identityService.identity = [];
+    identityService.identity.name = 'kyle';
+    backupService = new BackupService(identityService);
+  });
+  describe('Backup Codes', async () => {
+    it('generates one backup code', async () => {
+      await backupService.generateBackupCodes(1);
+      expect(backupService.backupCodes.length).to.eq(1);
+    });
```
Comment (marekkirejczyk) [inline universal-login-example/test/services/BackupService.js]: newline

File: universal-login-example/test/services/BackupService.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    it('generates more than one backup code', async () => {
+      await backupService.generateBackupCodes(2);
+      expect(backupService.backupCodes.length).to.eq(3);
+    });
```
Comment (marekkirejczyk) [inline universal-login-example/test/services/BackupService.js]: newline

File: universal-login-example/test/services/BackupService.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  before(async () => {
+    identityService = [];
+    identityService.identity = [];
+    identityService.identity.name = 'kyle';
+    backupService = new BackupService(identityService);
+  });
+  describe('Backup Codes', async () => {
+    it('generates one backup code', async () => {
+      await backupService.generateBackupCodes(1);
+      expect(backupService.backupCodes.length).to.eq(1);
```
Comment (marekkirejczyk) [inline universal-login-example/test/services/BackupService.js]: An expect on the backup code itself would be good. 
I would propose simple regexp to check content and length.

**Comment by Kyrrui** [comment]
@marekkirejczyk For some reason I can no longer merge in the changes after updating.

**Comment by alexvandesande** [comment]
@Kyrrui it might be in conflict with the recently merged https://github.com/EthWorks/UniversalLoginSDK/pull/100, as both made changes to approve connection screens. Have you checked the latest master?

**Comment by Kyrrui** [comment]
> @Kyrrui it might be in conflict with the recently merged #100, as both made changes to approve connection screens. Have you checked the latest master?

The branch is up to date, just needs a accepting review

**Pull request merged**